plugin-server/src/main/ingestion-queues/session-recording/process-event.ts

```diff
@@ -325,7 +329,8 @@ export const createSessionReplayEvent = (
         click_count: Math.trunc(clickCount),
         keypress_count: Math.trunc(keypressCount),
         mouse_activity_count: Math.trunc(mouseActivity),
-        first_url: url,
+        first_url: urls.length ? urls[0] : null,
+        urls: urls,
```

Comment:
from a clickhouse review perspective - we'll now add an array of URLs to each kafka message